pkg/controller/build/assets/Dockerfile.on-cluster-build-template

```diff
@@ -5,22 +5,22 @@
 # Decode and extract the MachineConfig from the gzipped ConfigMap and move it
```

Comment:
suggestion (non-blocking): Rename this file to `Containerfile.on-cluster-build-template` for consistency.